src/tsd/PutDataPointRpc.java

```diff
@@ -292,7 +293,18 @@ public void execute(final TSDB tsdb, final HttpQuery query)
     int queued = 0;
     final List<Deferred<Boolean>> deferreds = synchronous ? 
         new ArrayList<Deferred<Boolean>>(dps.size()) : null;
-    
+
+    if (tsdb.getConfig().enable_header_tag()) {
```

Comment:
There is actually no "tsd.http.enable_header_tag" config parameter. See other portion of patch, i add a method to check the !null of "tsd.http.header_tag" value.

Maybe i should check this is the constructor, like how "send_telnet_errors" is managed ?